core/src/scene/styleParam.cpp

```diff
@@ -159,7 +160,7 @@ StyleParam::Value StyleParam::parseString(StyleParamKey key, const std::string&
     case StyleParamKey::style:
         return _value;
     case StyleParamKey::font_size: {
-        float fontSize;
+        float fontSize = 10.f;
```

Comment:
This should just makes sure that fontSize is not used with an arbitrary (undefined) value